Client/game_sa/CModelInfoSA.cpp

```diff
@@ -1292,13 +1292,16 @@ void CModelInfoSA::SetColModel(CColModel* pColModel)
         // If no collision model has been set before, store the original in case we want to restore it
         if (!m_pOriginalColModelInterface)
             m_pOriginalColModelInterface = m_pInterface->pColModel;
+            m_pInterface->bAlphaTransparency = false;
```

Comment:
This statement is not inside the condition above. I assume you want it to be based on the indentation?

A bracketless condition will only execute the first statement directly after it. If you have multiple statements, you need brackets.

```suggestion
        if (!m_pOriginalColModelInterface)
        {
            m_pOriginalColModelInterface = m_pInterface->pColModel;
            m_pInterface->bAlphaTransparency = false;
        }
```

Reply:
Also just noticed, is this relevant to fixing the bug either?